tm2/pkg/bft/types/keys.go

```diff
@@ -1,6 +1,5 @@
 package types
 
 // UNSTABLE
-var (
-	PeerStateKey = "ConsensusReactor.peerState"
-)
```

Comment:
How this was not caught by the linters (more than 2 that check for this) is beyond me